src/BoundaryConditions/boundary_condition.jl

```diff
@@ -65,6 +65,22 @@ function BoundaryCondition(Classification::DataType, condition::Function;
     return BoundaryCondition(Classification(), condition)
 end
 
+function BoundaryCondition(Classification::RBCC, condition::Function;
+                           parameters = nothing,
+                           discrete_form = false,
+                           field_dependencies=())
+
+    if discrete_form
+        field_dependencies != () && error("Cannot set `field_dependencies` when `discrete_form=true`!")
+        condition = DiscreteBoundaryFunction(condition, parameters)
+    else
+        # The `ContinuousBoundaryFunction` is "regularized" for field location in the FieldBoundaryConditions constructor.
+        condition = ContinuousBoundaryFunction(condition, parameters, field_dependencies)
+    end
+
+    return BoundaryCondition(Classification, condition)
+end
```

Comment:
Isn't this almost copied verbatim from the function above?

Reply:
yeah this is implemented differently now